lib/apiservers/engine/backends/container.go

```diff
@@ -528,13 +528,6 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (contain
 
 	var err error
 
```

Comment:
If you remove the check for name conflict, what do we report when someone does docker create --name "..." that conflicts?

Reply:
We return the same error - it is returned from the `ReserveName` function: https://github.com/vmware/vic/blob/master/lib/apiservers/engine/backends/cache/container_cache.go#L178

The only difference between this error message string and the one in the check I'm removing is that `Conflict` is not capitalized in the former.